server/commands/ModelGeneration.py

```diff
@@ -0,0 +1,222 @@
+import bbi
+import os
+import sys
+import numpy as np
+import pandas as pd
+import requests
+import threading
+import utils.SlurmConfig as cfg
+
+
+def model(job):
+    data = job['data']
+    problem = data['problem']
+    trackInfo = data['data']
+    penalty = data['penalty']
+
+    dataPath = '%s%s/%s-%d-%d/' % (cfg.dataPath, trackInfo['name'], problem['ref'], problem['start'], problem['end'])
```

Comment:
repeated?

Reply:
Changed job handler so now it would be data['trackInfo'] instead of data['data']